pipeline/acquisition.py

```diff
@@ -0,0 +1,158 @@
+'''
+Schema of aquisition information.
+'''
+import datajoint as dj
+from pipeline import reference, subject
+
+schema = dj.schema('ttngu207_acquisition',locals())
```

Comment:
I know we haven't updated the documentation/tutorial yet but `dj.schema` no longer needs `locals()` :smiley: 
Also, let's not use user specific schema name. So in this case, something like

```suggestion
schema = dj.schema('hg2015_acquisition')
```
Now understandably, you wanted to prefix this with your username due to DB privilege. For cases like that, add `dj.config.get('database.prefix', '')` so that you can easily prepend phrases to all schema names during test etc:
```suggestion
schema = dj.schema(dj.config.get('database.prefix', '') + 'hg2015_acquisition')
```
This suggestion applies to all instance where you create a new schema.